apps/examples/testcase/le_tc/network/itc_net_setsockopt.c

```diff
@@ -223,12 +223,17 @@ int itc_net_setsockopt_main(void)
 
 	itc_net_setsockopt_n_invalid_opt_name(fd);
 	itc_net_setsockopt_p_rcvbuf(fd);
-	itc_net_setsockopt_p_reuseport(fd);
+	//itc_net_setsockopt_p_reuseport(fd);
```

Comment:
Why do you comment this? I can't find this function. If it is not existed, let's remove this line

Reply:
I commented out that functions.